doc/doc.sh

```
@@ -0,0 +1,125 @@
+#/bin/bash
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)